github/repos_projects.go

```diff
@@ -10,10 +10,19 @@ import (
 	"fmt"
 )
 
+// ProjectListOptions specifies the optional parameters to the
+// OrganizationsService.ListProjects and RepositoriesService.ListProjects methods.
+type ProjectListOptions struct {
+	// Indicates the state of the projects to return. Can be either open, closed, or all. Default: open
+	State string `url:"state,omitempty"`
```

Comment:
@shurcooL - do you know why there are many instances of
```
`url:"..."`
```
throughout this repo instead of
```
`json:"..."`
```
?

```bash
$ pt -l url:
github/activity_notifications.go
github/activity_star.go
github/gists.go
github/git_refs.go
github/issues_comments.go
github/issues.go
github/github.go
github/issues_milestones.go
github/orgs.go
github/orgs_outside_collaborators.go
github/orgs_members.go
github/orgs_teams.go
github/pulls_comments.go
github/pulls.go
github/repos_commits.go
github/repos_deployments.go
github/repos_forks.go
github/repos.go
github/repos_traffic.go
github/search.go
github/repos_contents.go
github/users.go
```

I just did a Google search and couldn't find any mention of `url:` for use with `encoding/json` and the [docs](https://golang.org/pkg/encoding/json/) don't seem to mention anything either.

In fact, reading through https://golang.org/pkg/encoding/json/#Marshal, it appears to me that all of these format strings are being completely ignored, which is probably not our intention.  :smile:

I'm thinking that we should probably change this one to
```
`json:"..."`
```
and then open a new issue to change the other ones... unless you point out what I'm missing.